Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:21.04 as builder
+WORKDIR /source
+COPY ./downloader.sh .
+RUN apt update && apt install -y wget curl && bash downloader.sh
```

Comment:
Does update work without `-y`?

Reply:
Yep tested before I pushed :)

Reply:
@ankitm123 lets get things working as is ATM as its just initial development, and improve with subsequent PRs once pipelines are setup